thinkhazard_processing.yaml

```diff
@@ -1,4 +1,37 @@
 sqlalchemy.url: postgresql://www-data:www-data@localhost:5432/thinkhazard_processing
-threshold: 98.0665
-return_periods: [2475, 475, 100]
+
+hazard_types:
+  FL:
+    hazard_type: flood
+    global:
+      return_periods:
+        LOW: 1000
+        MED: 50
+        HIG: 25
+    local:
+      return_periods:
+        LOW: 100
+        MED: 50
+        HIG: 25
+    threshold:
+      cm:
+      dm:
+      m:
+  EQ:
+    hazard_type: earthquake
+    global:
+      return_periods:
+        LOW: 2475
+        MED: 475
+        HIG: 250
+    local:
+      return_periods:
+        LOW: 2500
+        MED: 475
+        HIG: 100
+    threshold:
+      PGA-gal: 98.0665
```

Comment:
I would have prefered a "normalizing ratio" per unit, rather than redefining a new threshold per unit, but as seen with @arnaud-morvan we'll work on this later on.